libusb/hid.c

```diff
@@ -123,6 +123,10 @@ struct hid_device_ {
 #ifdef DETACH_KERNEL_DRIVER
 	int is_driver_detached;
 #endif
+
+	int error;
```

Comment:
I don't think we need to have it as a member here at all

Reply:
Not storing the error code would discard the error information returned by libusb.

In my case, value of this `int` is all I wanted. Ideally there would be an `hid_get_libusb_error_code` function.